includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -505,7 +504,15 @@ private function process_node( DOMElement $node ) {
 					array_unique(
 						array_map(
 							static function ( $validation_error ) {
-								unset( $validation_error['spec_name'] );
+								unset(
+									$validation_error['spec_name'],
+									// Remove other keys that may not not make the error unique.
```

Comment:
I'm removing any other keys that could potentially make the validation error unique (but it should not). Take for example this HTML element:

```html
<stop offset="5%" stop-color="gold" />
```

This would raise two separate `MANDATORY_TAG_ANCESTOR` validation errors, but each would have its own `required_ancestor_name` (`lineargradient` and `radialgradient`), making each validation error unique.